manual/source/examples/h5py/create_data.sh

```diff
@@ -2,26 +2,44 @@
 
 # (re)build all data files and text descriptions of data files
 
-#python BasicReader.py
-#python TestReader.py
-#python TestWriter.py
+ORIGINAL_WD=$(pwd)
+SCRIPT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-python BasicWriter.py
-punx tree prj_test.nexus.hdf5 > prj_test.nexus_structure.txt
-h5dump prj_test.nexus.hdf5 > prj_test.nexus_h5dump.txt
+PUNX_REMOVE_HEADER="+5"
 
-python externalExample.py
-punx tree external_angles.hdf5 > external_angles_structure.txt
+ROOT=$SCRIPT_ROOT/simple_example_basic
+cd $ROOT
+python $ROOT/simple_example_basic_write.py
+punx tree $ROOT/simple_example_basic.nexus.hdf5 | tail -n $PUNX_REMOVE_HEADER > $ROOT/simple_example_basic.nexus_structure.txt
+h5dump $ROOT/simple_example_basic.nexus.hdf5 > $ROOT/simple_example_basic.nexus_h5dump.txt
+python $ROOT/simple_example_basic_read.py
+
+ROOT=$SCRIPT_ROOT/simple_example_test
+cd $ROOT
+python simple_example_test_write.py
+python simple_example_test_read.py
+rm simple_example_test.nexus.hdf5
+
+ROOT=$SCRIPT_ROOT/simple_example_write1
+cd $ROOT
+python simple_example_write1.py
+punx tree simple_example_write1.hdf5 | tail -n $PUNX_REMOVE_HEADER > simple_example_write1_structure.txt
+h5dump simple_example_write1.hdf5 > simple_example_write1_h5dump.txt
+
+ROOT=$SCRIPT_ROOT/simple_example_write2
+cd $ROOT
+python simple_example_write2.py
+punx tree simple_example_write2.hdf5 | tail -n $PUNX_REMOVE_HEADER > simple_example_write2_structure.txt
+h5dump simple_example_write2.hdf5 > simple_example_write2_h5dump.txt
+
+ROOT=$SCRIPT_ROOT/external_example_write
+cd $ROOT
+python external_example_write.py
+punx tree external_angles.hdf5 | tail -n $PUNX_REMOVE_HEADER > external_angles_structure.txt
 h5dump external_angles.hdf5 > external_angles_h5dump.txt
-punx tree external_counts.hdf5 > external_counts_structure.txt
+punx tree external_counts.hdf5 | tail -n $PUNX_REMOVE_HEADER > external_counts_structure.txt
```

Comment:
Somehow, we need to become aware when punx no longer publishes this header.  Or refactor to count the number of lines to remove?

Reply:
Done: remove all lines that do not start with a space.